lib/ansible/plugins/callback/tree.py

```diff
@@ -9,6 +9,7 @@
     callback: tree
     callback_type: notification
     requirements:
+      - whitelist in configuration
```

Comment:
this is used even w/o whitelisting when the `--tree` option is used, so not always a requirement